python/tests/test_search_disk_index.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+import time
+import argparse
+import numpy as np
+import diskannpy
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('query_path', type=str, help='Path to the input query set of vectors.')
+parser.add_argument('ground_truth_path', type=str, help='Path to the input groundtruth set.')
+parser.add_argument('index_path_prefix', type=str, help='Path prefix for index files.')
+parser.add_argument('output_path_prefix', type=str, help='Prefix for the generated output files.')
+parser.add_argument('K', type=int, help='k value for recall@K.')
+parser.add_argument('W', type=int, help='Beamwidth for search.')
+parser.add_argument('T', type=int, help='Number of threads to use for search.')
+parser.add_argument('C', type=int, help='Number of nodes to cache for search.')
+
+
+args = parser.parse_args()
+
+recall_at = args.K
+W = args.W
+# Use multi-threaded search only for batch mode.
+num_threads = args.T
+num_nodes_to_cache = args.C
+single_query_mode = False
+l_search = [40, 50, 60, 70, 80, 90, 100, 110, 120]
```

Comment:
Why not allow the user to set these, and default to this array if the user doesnt want to set it.